windows.md

```diff
@@ -328,7 +328,15 @@ Before you start, please make sure that you're running Windows 10 or Windows 11.
 
     <img src="windows-7-virtualisation-enabled.png"><br><br>
 
-    If virtualization is not enabled, [enable VT-x or SVM in your UEFI or BIOS](https://example.com/add-link-here).
+    Only if virtualization is not enabled, **Enable VT-x or SVM in your UEFI or BIOS**
```

Comment:
we can leave this as it was before (remove "Only" and remove the bold, add a colon at the end of the line)

Reply:
e2aff03cb9f0c7ed1b735fd0c8eabb0e11148841